File: bronze/4344/python/main.py

if (__name__ == "__main__"):
    num = int(input())
    for i in range(num):
        arr = list(map(int, input().split()))
        size = arr.pop(0)
        score = (float)(sum(arr)) / size
        count = 0
        for j in range(size):
            if (arr[j] > score):
                count += 1
        result = (float)(count) / size * 100
        print(f"{result:.3f}%")

File: bronze/4344/c++/main.cpp
#include <iostream>

int main()
{
    int num;
    std::cin >> num;
    int n, temp, count;
    float score, result;
    for (int i = 0; i < num; i++) {
        std::cin >> n;
        score = 0.0f;
        count = 0;
        int arr[n];
        for (int j = 0; j < n; j++) {
            std::cin >> arr[j];
            score += arr[j];
        }
        score /= n;
        count = 0;
        for (int j = 0; j < n; j++) {
            if (arr[j] > score) {
                count++;
            }
        }
        result = (float)count / n * 100;
        printf("%.3f%\n", result);
        
    }

    return 0;
}